data-analysis/graphs.py

```diff
@@ -0,0 +1,183 @@
+import matplotlib.colors as clr
+import matplotlib.cm as cm
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+import squarify 
+
+def init_setup():
+
+    sns.set(rc={'figure.figsize':(15,8)})
+    sns.set(style="white", context="talk")
+   
+    
+def genre_pie(genre_counts, genre_list, name):
```

Comment:
Add documentation for all of these functions: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html